sched_slack_bot/views/schedule_dialog.py

```diff
@@ -62,7 +62,7 @@ def get_display_name_input(schedule: Optional[Schedule] = None) -> InputBlockWit
     return InputBlockWithBlockId(
         label="Display Name",
         hint=PlainTextObject(text="Name for your rotating schedule"),
-        element=PlainTextInputElement(initial_value=initial_value),
+        element=PlainTextInputElement(initial_value=initial_value, max_length=99),
```

Comment:
does this give the user some feedback on trying to enter too long names? If not we should probably adjust the hint to say (max 99 characters) or something. 

But maybe good to adjust the hint either way, if it looks ok :)

Reply:
![image](https://github.com/user-attachments/assets/1b5d7d68-fff0-4a3c-a94a-2975eca371ef)

![image](https://github.com/user-attachments/assets/8622de5e-7dc4-48c2-86a9-57e633800f80)